src/bin/generate_rr_bracket.rs

```diff
@@ -8,23 +9,48 @@ extern crate dotenv;
 
 // Generates a 4-player round robin bracket
 #[tokio::main]
-async fn main() {
-    dotenv::dotenv().unwrap();
-    let mut db = raw_diesel_cxn_from_env().unwrap();
+async fn main() -> anyhow::Result<()> {
+    #[cfg(feature = "development")]
```

Comment:
I have this gated on a feature flag because I dont want `Season::ensure_started_season` to be callable from app code